redfish-core/lib/systems.hpp

```diff
@@ -2281,11 +2281,7 @@ inline void getChapData(const std::shared_ptr<bmcweb::AsyncResp>& asyncResp)
                     const dbus::utility::DBusPropertiesMap& propertiesList) {
         if (ec)
         {
-            if (ec.value() != EBADR)
-            {
-                BMCWEB_LOG_ERROR("Get ChapData D-bus error: {}", ec.value());
-                messages::internalError(asyncResp->res);
-            }
+            // ChapData isn't that important. Ignore all errors.
```

Comment:
Can we leave a DEBUG message here?
In the future (when bmcweb dynamic logging is included), this part can be debugged if needed.

Reply:
Done. I left the logging in but moved it to DEBUG